css/media.css

```diff
@@ -0,0 +1,1039 @@
+    padding: 0 0;
+    margin: 0 8px 0 0;
+  }
+
+  .products__product-card:nth-last-child(-n+2) {
+    display: none;
+  }
+
+  .product-card__picture {
+    margin: 0 0 28px 13px;
+  }
+
+  .product-card__picture img {
+    width: 234px;
+    height: 219px;
+  }
+
+  .product-card__name {
+    font-size: 20px;
+    margin: 0 0 13px 0;
+  }
+
+  .product-card__price {
+    margin-right: 58px;
+    font-size: 25px;
+  }
+
+  .product-card::after {
+    display: none;
+  }
+
+  .circles {
+    margin-right: 11px;
+  }
+
+  .circles__link {
+    width: 20px;
+    height: 20px;
+    margin-left: 6px;
+  }
+
+  .color-option {
+    padding: 0;
+  }
+
+  /* promo-section */
+
+  .promo-section {
+    position: relative;
+    padding: 0 0 124px 0;
+    background-color: #000;
+    color: #fff;
+  }
+
+  .promo-section::before {
+    content: '';
+    position: absolute;
+    top: -35px;
+    z-index: 2;
+    width: 100%;
+    height: 55px;
+    transform: skewY(2.6deg);
+    background-color: #000;
+  }
+
+  .promo-section::after {
+    content: '';
+    position: absolute;
+    bottom: -17px;
+    z-index: 0;
+    width: 100%;
+    height: 352px;
+    transform: skewY(2.6deg);
+    background-color: #fff;
+  }
+
+  .video-block {
+    flex-direction: column;
+    padding: 0;
+    min-height: 641px;
+  }
+
+  .video-block__text-wrapper {
+    max-width: 100%;
+    margin-top: 0;
+    padding: 0 30px;
+    text-align: center;
+    order: -1;
+  }
+
+  .video-block__slogan {
+    margin-top: 34px;
+    font-size: 30px;
+    line-height: 37px
+  }
+
+  .video-block__text {
+    max-width: 99%;
+    margin: 14px 0 27px 5px;
+    line-height: 25.47px;
+    font-size: 20px;
+  }
+
+  .video-block__link {
+    display: none;
+  }
+
+  .video-block__picture-wrapper {
+    margin-right: 0;
+  }
+
+  .video-block__pic {
+    /* width: 698px; */
+    display: block;
+    width: 91%;
+    margin: 0 auto;
+  }
+
+  .promo-section__subtitle {
+    margin: 42px 0 43px 0;
+    /* margin-top: 42px; */
+  }
+
+  .stories {
+    justify-content: space-between;
+  }
+
+  .promo-section__stories {
+    max-width: 680px;
+    margin: 0 auto;
+  }
+
+  .stories-card {
+    max-width: 312px;
+  }
+
+  .stories-card__pic {
+    max-width: 100%;
+    min-height: 166px;
+    /* min-height: 100%; */
+    /* object-fit: contain; */
+  }
+
+  .stories-card__title {
+    /* margin-bottom: 8px; */
+    margin: 25px 0 8px 0;
+    font-size: 25px;
+  }
+
+  .stories-card__descr {
+    margin-bottom: 37px;
+    font-size: 20px;
+    line-height: 25px;
+  }
+
+  .stories-card__wrapper {
+    align-items: center;
+  }
+
+  .stories-card__link-more {
+    font-size: 20px;
+    color: #fff;
+  }
+
+  .stories-card__link-more::after {
+    content: url(../img/Arrow-768-white.png);
+    margin-left: 13px;
+  }
+
+  .stories-card__link-more:visited {
+    color: #fff;
+  }
+
+  .stories-card__publication-date {
+    font-size: 15px;
+    color: #D1D1D1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#D1D1D1" to be "#d1d1d1" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Axion2/pullRequest?prid=5654145)